src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java

```diff
@@ -171,14 +171,6 @@ public RightClickMenu(JabRefFrame frame, BasePanel panel) {
             }
         });
 
-        add(new GeneralAction(Actions.MERGE_DOI, Localization.lang("Get BibTeX data from DOI")) {
```

Comment:
No worries. :) Action in right click menu stayed in there. :trophy: 

The right click menu now only contains actions related to selected entries in the table. Hence, the "new from doi" was moved to the menu.